### PR TITLE
adds in blog post about node scaffolding

### PR DESCRIPTION
**Pull request by nellshamrell**
Signed-off-by: Nell Shamrell <nellshamrell@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by fnichol** [comment]
Sorry about the long build, I'm restarting the Travis job and you should be good again.

File: www/source/blog/2017-10-09-node-scaffolding-habitat.html.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+date: 2017-10-09
+author: Nell Shamrell-Harrington
+tags: scaffolding
+category: community
+classes: body-article
+---
+
+Greetings fellow Habicats!
+
+One of the most useful feature of Habitat is scaffoldings - these allow you to package your application with Habitat with very minimal code.  They enable you to very rapidly (within minutes) package an application and export it to whatever format you need.  One of the most popular Habitat scaffoldings is the NodeJS scaffolding.
```
Comment (reset) [inline www/source/blog/2017-10-09-node-scaffolding-habitat.html.md]: feature -> features

Comment (reset) [reply www/source/blog/2017-10-09-node-scaffolding-habitat.html.md]: Just `scaffolding` is the plural we've been going with for Scaffolding

Comment (reset) [inline www/source/blog/2017-10-09-node-scaffolding-habitat.html.md]: I'd stick with uppercasing Scaffolding as it's a proper noun in Habitat

Comment (reset) [inline www/source/blog/2017-10-09-node-scaffolding-habitat.html.md]: I'd recommend highlighting that Scaffolding makes things _even easier_ than packaging an app without it. It's also a good time to call out that Scaffolding is made available to encapsulate in a 1:1 way how to package an app for a given language runtime (if applicable) and build tool

File: www/source/blog/2017-10-09-node-scaffolding-habitat.html.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+If you'd like, start the app locally
+
+```console
+$ npm start
+```
+
+Then check out your running application at http://localhost:3000/
+
+## Habiterizing your NodeJS application
```
Comment (reset) [inline www/source/blog/2017-10-09-node-scaffolding-habitat.html.md]: Marketing thinks we should be going with "Habitizing" for this made up word ;)

File: www/source/blog/2017-10-09-node-scaffolding-habitat.html.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+You can use version numbers in these formats "5.0.0", "v5.0.0", "=5.0.0", ">=5.0.0", ">5.0.0","<=5.0.0","<5.0.0"
+
+* Write an .nvmrc in your application's root directory
+* Set the scaffolding_node_pkg variable in your Plan with a valid Habitat package identifier (i.e. your_origin/node/5.0.0)
+
+What about if your application specifies different versions in different places?  A Plan variable will take priority ov package.json and package.sjon will take priority over an .nvmrc file.
+
+
+## Coming Soon
```
Comment (reset) [inline www/source/blog/2017-10-09-node-scaffolding-habitat.html.md]: I love this section

**Comment by reset** [comment]
@nellshamrell great article! Some minor nit picks inline and suggestions which aren't a showstopper. It also looks like you've double spaced after every period - it's not a hill I'm ready to die on, but with the wide availability of proportional fonts vs monospacing on typewriters the norm is now a single space 🚲  🏠  😄

**Comment by nellshamrell** [comment]
Thanks for the review, @reset!  Indeed, the double space thing is just too ingrained in my fingers from years of writing academic papers, so it's something I need to be reminded of :)

Closing this for now because Marketing has asked me to add some Builder content in, which I will do, then re-open when it is done :)

**Pull request closed without merging**